third_party/txt/src/txt/text_style.cc

```diff
@@ -53,6 +53,11 @@ bool TextStyle::equals(const TextStyle& other) const {
     return false;
   if (text_shadows.size() != other.text_shadows.size())
     return false;
+  for (size_t font_index = 0; font_index < font_family_fallback.size();
```

Comment:
Add a check that `font_family_fallback.size()` matches `other.font_family_fallback.size()`

Also, move this so it is not between the checks for `text_shadows`